mod_reforged/msu_systems/mod_settings.nut

```diff
@@ -13,3 +13,10 @@ tacticalTooltipPage.addRangeSetting("CollapseActivesWhenX", 5, 0, 20, 1, "Collap
 tacticalTooltipPage.addBooleanSetting("TacticalTooltip_CollapseAsText", false, "Collapse as Text", "If enabled, then skills collapse using their names as text, otherwise they collapse using their icons.");
 tacticalTooltipPage.addBooleanSetting("ShowStatusPerkAndEffect", true, "Show Status Perk And Effect", "Some Perks are also Status Effects. Usually their Effect is hidden until some condition is fulfilled. When this setting is enabled, these perks show up in the Perks category even when they show up under Effects (e.g. when their effect is active). When disabled, when they appear under Effects, they will be hidden from the Perks category. This can help save space on the tooltip.");
 tacticalTooltipPage.addBooleanSetting("HeaderForEmptyCategories", false, "Show Header for empty categories");
+
+::Reforged.Mod.Keybinds.addSQKeybind("Tactical_WaitRound", "ctrl+space", ::MSU.Key.State.Tactical, function()
```

Comment:
As discussed on Discord, `Ctrl+Space` is not a good keybind to have due to conflicts with Windows. I suggested the use of `Alt+R` because `R` is already the end-round button and having an `alt` modifier for such keys to do something which is similar in theme seems like a good design. Other choices such as `H` or `G` are keys which could potentially be used for a more frequently used thing, because Wait-All is a very niche use case.

Reply:
I tend towards **H**

Reply:
Let's finalize this and go with `Alt+R` for now.

Reply:
Alt is the hotkey for toggling the entity overlays though.
And from my tests that overlay is toggled every time I let go of my Alt key. Even if I suppress vanilla keybinds via MSU setting.